webapi/cvmfsmon_api.py

```diff
@@ -109,15 +112,21 @@ def parse_api_conf():
                     excludes.append(words[1])
                 elif words[0] == 'disabletest':
                     disables.append(words[1])
+		elif words[0] == 'updated-slowrepos':
```

Comment:
likewise the keyword here should still be singular.

Reply:
Very sorry for the errors, I noticed what I had done just as I was about to reply. Fixing it now!